data/json/items/melee/bludgeons.json

```diff
@@ -867,19 +886,19 @@
   },
   {
     "type": "GENERIC",
-    "id": "sharp_toothbrush",
+    "id": { "str": "sharp_toothbrush" },
     "symbol": ",",
     "color": "light_blue",
-    "name": { "str": "sharpened toothbrush", "str_pl": "sharpened toothbrushes" },
-    "description": "A personal hygiene item turned into a shank.",
+    "name": { "str": "plastic shank", "str_pl": "plastic shanks" },
+    "description": "An oblong plastic trinket with the far end ground down into a stabbing point.",
     "category": "weapons",
     "price": 400,
+    "price_postapoc": 0,
```

Comment:
Accuracy is
base -2
grip: solid +1
length: none 0
surface: point -2
balance: neutral +0
for a total of -3

Somewhere in here, you should add `"to_hit": -3`